test/fgpu/test_engine.py

```diff
@@ -945,7 +944,7 @@ async def fill_in(self) -> InItem | None:
             if self._in_item is None:
                 nonlocal counter
                 counter += 1
-                if counter == 6:
+                if counter == 12:
```

Comment:
I assume this just means "wait longer before checking the `steady-state-timestamp` (?)